docs/docsite/rst/guide_aws.rst

```diff
@@ -0,0 +1,281 @@
+.. _ansible_collections.amazon.aws.docsite.aws_intro:
+
+Amazon Web Services Guide
+=========================
+
+Introduction
+````````````
+
+Ansible contains a number of modules for controlling Amazon Web Services (AWS).  The purpose of this
+section is to explain how to put Ansible modules together (and use inventory scripts) to use Ansible in AWS context.
+
+Requirements for the AWS modules are minimal.
+
+All of the modules require and are tested against recent versions of boto, usually boto3. Check the module documentation for the minimum required version for each module. You must have the boto3 Python module installed on your control machine. You may also need the original boto package. You can install these modules from your OS distribution or using the python package installer: ``pip install boto3``.
+
+Whereas classically Ansible will execute tasks in its host loop against multiple remote machines, most cloud-control steps occur on your local machine with reference to the regions to control.
+
+In your playbook steps we'll typically be using the following pattern for provisioning steps::
+
+    - hosts: localhost
+      gather_facts: False
+      tasks:
+        - ...
+
+.. _ansible_collections.amazon.aws.docsite.aws_authentication:
+
+Authentication
+``````````````
+
+Authentication with the AWS-related modules is handled by either
+specifying your access and secret key as ENV variables or module arguments.
+
+For environment variables::
+
+    export AWS_ACCESS_KEY_ID='AK123'
+    export AWS_SECRET_ACCESS_KEY='abc123'
+
+For storing these in a vars_file, ideally encrypted with ansible-vault::
+
+    ---
+    ec2_access_key: "--REMOVED--"
+    ec2_secret_key: "--REMOVED--"
+
+Note that if you store your credentials in vars_file, you need to refer to them in each AWS-module. For example::
+
+    - ec2
+      aws_access_key: "{{ec2_access_key}}"
+      aws_secret_key: "{{ec2_secret_key}}"
+      image: "..."
```

Comment:
I guess we'll want to update this to recommend the use of module_defaults?

Reply:
Thanks for spotting that @tremble! The guide will need sundry updates to the content. The purpose of this PR is mainly to get it moved, and to serve as an example of how to move other Scenario Guides.

Reply:
@tremble - Ignore my comment - I just saw your overall review comment . . .